resources/ext/config/logback.xml

```diff
@@ -20,16 +20,9 @@
         </encoder>
     </appender>
 
-    <!-- Supress internal ActiveMQ logging -->
-    <logger name="org.apache.activemq" level="warn"/>
-
     <!-- Supress internal Spring Framework logging -->
     <logger name="org.springframework.jms.connection" level="warn"/>
```

Comment:
I don't see Spring Framework in our dep tree anymore, can this be removed too?

Reply:
Not sure -- if it's not anywhere in the dep tree, possibly, though wonder if it might be loaded dynamically.  I can do some digging -- see why it was added in the first place.